tests/unit/s2n_connection_serialize_test.c

```diff
@@ -213,8 +219,7 @@ int main(int argc, char **argv)
 
             uint8_t cipher_suite[S2N_TLS_CIPHER_SUITE_LEN] = { 0 };
             EXPECT_SUCCESS(s2n_stuffer_read_bytes(&stuffer, cipher_suite, S2N_TLS_CIPHER_SUITE_LEN));
-            EXPECT_BYTEARRAY_EQUAL(cipher_suite, server_conn->secure->cipher_suite->iana_value,
-                    S2N_TLS_CIPHER_SUITE_LEN);
+            EXPECT_BYTEARRAY_EQUAL(cipher_suite, iana_value, S2N_TLS_CIPHER_SUITE_LEN);
```

Comment:
correct. I have to move around some stuff so that I get the information off the connection before calling serialize, as serialize now preforms a wipe.